ddtrace/contrib/aiohttp/middlewares.py

```diff
@@ -9,9 +11,14 @@
 REQUEST_CONTEXT_KEY = 'datadog_context'
 REQUEST_SPAN_KEY = '__datadog_request_span'
 
+PARENT_TRACE_HEADER_ID = 'x-ddtrace-parent_trace_id'
+PARENT_SPAN_HEADER_ID = 'x-ddtrace-parent_span_id'
+
+_SPAN_MIN_ERROR = 400
+
 
 @asyncio.coroutine
-def trace_middleware(app, handler):
+def trace_middleware(app, handler, enable_distributed=False):
```

Comment:
since we're already using configs here, what do you think if we put `enable_distributed` as a config object instead of having it as a kwarg here? so you can do:
```python
distributed_tracing = app[CONFIG_KEY]['enable_distributed_tracing']
```

and users can set that in the application config